service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderControllerTest.java

```diff
@@ -149,30 +156,36 @@ void midEventShouldAddDocumentReferenceToOrderAction() throws Exception {
                 .build());
     }
 
+    //TODO: test is failing on nextHearingDateLabel assertion.
```

Comment:
Left you a comment on slack Sam but this fails because you've removed the following from CaseDataGeneratorHelper 

`put("cmoToAction", CaseManagementOrder.builder()
                .nextHearing(NextHearing.builder()
                    .id(fromString("ecac3668-8fa6-4ba0-8894-2114601a3e31"))
                    .build())
                .build())
            .build();

caseDataGeneratorHelper`

Reply:
nope, it's there 
```
  .put("cmoToAction", CaseManagementOrder.builder()
                .nextHearing(NextHearing.builder()
                    .id(fromString("ecac3668-8fa6-4ba0-8894-2114601a3e31"))
                    .build())
                .build())
            .build();
```

Reply:
I have removed the dependency on this becsause I hate that service